Pod/Classes/WPInputMediaPickerViewController.m

```diff
@@ -51,10 +51,30 @@ - (void)setupMediaPickerViewController {
     [self overridePickerTraits];
     
     self.mediaPicker.view.frame = self.view.bounds;
-    self.mediaPicker.view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;    
+    self.mediaPicker.view.translatesAutoresizingMaskIntoConstraints = NO;
     [self.view addSubview:self.mediaPicker.view];
+    if (@available(iOS 11.0, *)) {
+        UILayoutGuide *layoutGuide = self.view.safeAreaLayoutGuide;
+        [NSLayoutConstraint activateConstraints:
+         @[
+           [self.mediaPicker.view.leadingAnchor constraintEqualToAnchor:layoutGuide.leadingAnchor constant:0],
+           [self.mediaPicker.view.trailingAnchor constraintEqualToAnchor:layoutGuide.trailingAnchor constant:0],
+           [self.mediaPicker.view.topAnchor constraintEqualToAnchor:layoutGuide.topAnchor constant:0],
```

Comment:
I think we need to continue to use the view's top and bottom anchors, otherwise there's an issue where the collection view never scrolls behind the home indicator (content should flow behind it):

![simulator screen shot - iphone x - 2017-09-25 at 17 43 07](https://user-images.githubusercontent.com/4780/30821389-1bb4317e-a21d-11e7-974c-4159ae94cf55.png)

We could clean this section up by doing something like this:

```
NSLayoutAnchor *leadingAnchor = self.view.leadingAnchor;
NSLayoutAnchor *trailingAnchor = self.view.trailingAnchor;

if (@available(iOS 11.0, *)) {
    leadingAnchor = self.view.safeAreaLayoutGuide.leadingAnchor;
    trailingAnchor = self.view.safeAreaLayoutGuide.trailingAnchor;
}

[NSLayoutConstraint activateConstraints: @[
   [self.mediaPicker.view.leadingAnchor constraintEqualToAnchor:leadingAnchor constant:0],
   [self.mediaPicker.view.trailingAnchor constraintEqualToAnchor:trailingAnchor constant:0],
   [self.mediaPicker.view.topAnchor constraintEqualToAnchor:self.view.topAnchor constant:0],
   [self.mediaPicker.view.bottomAnchor constraintEqualToAnchor:self.view.bottomAnchor constant:0]
   ]];
```

Reply:
The issue with that is when you have a horizontal scroll....